src/libs/elektra/kdb.c

```diff
@@ -1885,6 +1871,17 @@ int kdbGet (KDB * handle, KeySet * ks, Key * parentKey)
 		goto error;
 	}
 
+	/* TODO (kodebach): implement actual steps with new global plugins
```

Comment:
Is this a marker for @atmaxinger?

Reply:
Actually, I just moved this down, because before it went "Step 14" then "Step 13". In fact this part is already gone because of #4495.